.eslintrc.yml

```diff
@@ -100,6 +100,7 @@ overrides:
       no-shadow: off
       '@typescript-eslint/no-shadow': warn
       '@typescript-eslint/no-explicit-any': warn
+      '@typescript-eslint/no-unused-vars': ["error", { "ignoreRestSiblings": true }]
```

Comment:
double quotes and space inside curly brackets?! :P

Reply:
All quotes matter!! 😄